src/requirements/admin/track-users.ts

```diff
@@ -32,7 +32,7 @@ function isOld(semester: FirestoreSemester) {
   if (semester.year < year) {
     return true;
   }
-  if (seasonToMonth(semester.season) <= month) {
+  if (semester.season && seasonToMonth(semester.season) <= month) {
```

Comment:
@willespencer can we investigate why/where `season` is a missing property? This check shouldn't be necessary.

Reply:
Can you actually add a comment for what this function is? Idk what old means and what it's referring to

Reply:
@benjamin-shen investigated and talked offline - at least one dev user still does not have a season property, but all prod users do. Solution is to re-run the script on dev.